Dockerfile

```diff
@@ -128,7 +128,7 @@ RUN if [ "$OPENVINO_TOOLKIT" = "yes" ]; then \
 # TODO: CHANGE URL
 ARG WITH_DEXTR
 ENV WITH_DEXTR=${WITH_DEXTR}
-ENV DEXTR_MODEL_DIR=${HOME}/models/dextr
+ENV DEXTR_MODEL_DIR=${HOME}/dextr
```

Comment:
@bsekachev , can we fix the problem inside our internal infrastructure? The change is looking very strange.

Reply:
@nmanovic 
It is not our internal infrastructure. It is also [public](https://github.com/opencv/cvat/blob/923bcacd7784091b44548a6473ffa50e4193d841/docker-compose.yml#L58).